packages/ui/src/components/Button/index.tsx

```diff
@@ -251,6 +251,7 @@ type CommonProps = {
   disabled?: boolean
   iconPosition?: 'left' | 'right'
   iconVariant?: ComponentProps<typeof Icon>['variant']
+  iconTitle?: string
```

Comment:
I think aria-label is enough already for having a button with icon only

Reply:
removed until I get a reply form numa https://github.com/scaleway/ultraviolet/pull/4077/commits/d7db7f3173f342dd6bf57bff21c80c0cefba6c22